docs/roles/integrator/exchange-integration.md

```diff
@@ -64,18 +63,17 @@ Your response should look like this:
 
 **Note:** Gas prices can change between blocks. Even for transactions with deterministic gas cost, the cost in NEAR could also be different.
 
-## Account
+### Accounts
+
+Please see the [documentation for accounts](/docs/concepts/account) for basic information.
 
-- We support [implicit account](https://nomicon.io/DataStructures/Account.html#implicit-account-ids) creation which allows exchanges to create accounts without paying for transactions.
+- For exchanges, NEAR supports [implicit account](https://nomicon.io/DataStructures/Account.html#implicit-account-ids) creation which allows the creation of accounts without paying for transactions.
 - You can create an implicit account by following the steps in [this guide](/docs/roles/integrator/implicit-accounts).
-- Accounts must have enough tokens cover its storage. Storage cost per byte is 0.0001 NEAR and an account with one
-  access key must maintain a balance of at least 0.0182 NEAR. For more details, see [this section of the economics paper](https://near.org/papers/economics-in-sharded-blockchain/#transaction-and-storage-fees)
+- Accounts must have enough tokens cover its storage. Storage cost per byte is 0.0001 NEAR, and an account with one access key must maintain a balance of at least 0.0182 NEAR. For more details, see [this section of the economics paper](https://near.org/papers/economics-in-sharded-blockchain/#transaction-and-storage-fees).
```

Comment:
"typically costs"? I thought this was added by Bowen because it's precise. I think this is a constant, and don't want to add ambiguity

Reply:
Storage price actually can change but will never increase (as it would break a bunch of accounts) but can decrease if the price of NEAR goes up significantly and we want to do another 10x reduction in cost. I should change this wording from "typically" to "currently".